packaging_automation/tests/files/citus.spec

```diff
@@ -81,6 +84,66 @@ echo %{pginstdir}/share/extension/%{sname}.control >> installation_files.list
 %doc %{pginstdir}/doc/extension/NOTICE-%{sname}
 
 %changelog
+* Tue Feb 01 2022 - Gurkan Indibay <gindibay@microsoft.com> 10.1.4.citus-1
+- Official 10.1.4 release of Citus
```

Comment:
Do we have missing entries for `10.2.4` here?

Is this not covered by the tests somehow?

rpmbuild may error out on this spec due to the mismatch between the `Version:` field and the first version in changelog entry. I am not 100% sure, but I recall observing some failures like this one in the past.